workshops/hcload/README.md

```diff
@@ -0,0 +1,465 @@
+---
+name: Hackclub CDN Uploader
+description: Make a CLI to upload to the Hackclub CDN and implement uploading to the CDN in your apps
+author: '@KhushrajRathod'
+img: https://cloud-bb4wyl0oo.vercel.app/0screenshot_2020-11-11_at_10.00.26_pm.png
+---
+
+# Hackclub CDN Uploader 
+
+<video autoplay loop muted>
```

Comment:
GitHub + the Workshops website don't support <video> tags, so this demo doesn't appear. Can you turn it into a gif or some format that is accessible in the workshop?